test/integration/sagemaker/test_elastic_inference.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License").
+#  You may not use this file except in compliance with the License.
+#  A copy of the License is located at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  or in the "license" file accompanying this file. This file is distributed
+#  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+#  express or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+import logging
+import os
+
+import numpy as np
+import pytest
+from sagemaker.mxnet import MXNetModel
+from sagemaker.utils import sagemaker_timestamp
+
+from test.integration import RESOURCE_PATH, EI_SUPPORTED_REGIONS
+from test.integration.sagemaker.timeout import timeout_and_delete_endpoint_by_name
+
+logger = logging.getLogger(__name__)
+logging.getLogger('boto3').setLevel(logging.INFO)
+logging.getLogger('botocore').setLevel(logging.INFO)
+logging.getLogger('factory.py').setLevel(logging.INFO)
+logging.getLogger('auth.py').setLevel(logging.INFO)
+logging.getLogger('connectionpool.py').setLevel(logging.INFO)
+logging.getLogger('session.py').setLevel(logging.DEBUG)
+logging.getLogger('sagemaker').setLevel(logging.DEBUG)
+
+
+@pytest.fixture(autouse=True)
+def skip_if_no_accelerator(accelerator_type):
+    if accelerator_type is None:
+        pytest.skip('Skipping because accelerator type was not provided')
+
+
+@pytest.fixture(autouse=True)
+def skip_if_non_supported_ei_region(region):
+    if region not in EI_SUPPORTED_REGIONS:
+        pytest.skip('EI is not supported in {}'.format(region))
+
+
+@pytest.fixture
+def pretrained_model_data(region):
+    return 's3://sagemaker-sample-data-{}/mxnet/model/resnet/resnet_50.tar.gz'.format(region)
```

Comment:
just want to make sure - has this been uploaded in all the expected regions already?

Reply:
aws s3 ls s3://sagemaker-sample-data-us-west-1/mxnet/model/resnet/resnet_50.tar.gz
2019-02-15 15:05:25  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-us-east-1/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:09:56  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-us-east-2/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:10:12  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-eu-west-1/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:10:26  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-eu-west-2/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:10:57  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-eu-central-1/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:11:10  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-ca-central-1/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:11:29  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-ap-northeast-2/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:11:40  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-ap-northeast-1/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:11:56  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-ap-south-1/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:12:23  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-ap-southeast-1/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:13:15  225557606 resnet_50.tar.gz
aws s3 ls s3://sagemaker-sample-data-ap-southeast-2/mxnet/model/resnet/resnet_50.tar.gz
2018-11-15 15:13:42  225557606 resnet_50.tar.gz

Reply:
it isn't in pdt